backend/src/main/java/com/google/minicrm/servlets/SettingsServlet.java

```diff
@@ -2,21 +2,19 @@
 
 
 import com.google.appengine.api.datastore.*;
```

Comment:
nit: avoid * import (I can go ahead and auto-fix this with IntelliJ if you want)

Reply:
Yeah, can you quickly fix this